enable/drag.py

```diff
@@ -156,18 +156,18 @@ def drag_done(self, event):
         start_event = self.start_event
 
         # 'Unhook' the drag done notification handler:
-        self.on_trait_change(self.drag_done, self.drag_event, remove=True)
+        self.observe(self.drag_done, self.drag_event, remove=True)
 
         # Compute the new drag bounds:
-        x = event.x
-        y = event.y
+        x = event.new.x
+        y = event.new.y
```

Comment:
This looks a bit sketchy. Like `event` is actually a `DragEvent` instance. So adding observe here makes things very confusing.

Reply:
`event.new` is the `DragEvent` instance, i was going to do `drag_event = event.new` and propagate the change down, but `self.drag_event` already exists.  Maybe `new_drag_event`?  

I agree this is very confusing, I will go back over this code